src/anemoi/models/layers/block.py

```diff
@@ -316,15 +330,20 @@ def __init__(
 
         self.num_chunks = num_chunks
 
-        self.lin_key = nn.Linear(in_channels, num_heads * self.out_channels_conv)
-        self.lin_query = nn.Linear(in_channels, num_heads * self.out_channels_conv)
-        self.lin_value = nn.Linear(in_channels, num_heads * self.out_channels_conv)
-        self.lin_self = nn.Linear(in_channels, num_heads * self.out_channels_conv, bias=bias)
-        self.lin_edge = nn.Linear(edge_dim, num_heads * self.out_channels_conv)  # , bias=False)
+        # Uses the implementation defined in config.model.layer_kernels.<kernel>
+        # (unless it is not availible, in which case it will fall back to torch.nn.<kernel>)
+        Linear = layer_kernels["Linear"]
```

Comment:
are these keys guaranteed to be in the `layer_kernels` dict?

Reply:
possible alternative:

```python3
Linear = layer_kernels.get("Linear", torch.nn.Linear)
```

Reply:
Oh that's nice! I just pushed a version which uses torch.nn if nothing is set. But I'll replace it with that

Reply:
done